modules/local/deseq2/differential_expression/main.nf

```diff
@@ -0,0 +1,55 @@
+process DESEQ2_DIFFERENTIAL_EXPRESSION {
```

Comment:
I am sure there is potential for this module with the new DA pipeline to benefit from each other :)

Reply:
🙌🏼